ext/fileinfo/fileinfo.c

```diff
@@ -342,6 +342,10 @@ PHP_FUNCTION(finfo_buffer)
 		RETURN_THROWS();
 	}
 
+	if (ZEND_NUM_ARGS() == 4 || (hasThis() && ZEND_NUM_ARGS() == 3)) {
+		php_error_docref(NULL, E_DEPRECATED, "The $context parameter has no effect for finfo_buffer()");
```

Comment:
Contrary to the other deprecations, this does not abort on `EG(exception)`. Is that intentional?

Reply:
Nope forgot, will fix and probably split them up into different PRs.